homeassistant/components/overkiz/climate_entities/atlantic_pass_apc_heating_and_cooling_controlled_zone.py

```diff
@@ -0,0 +1,248 @@
+"""Support for Atlantic Pass APC Heating Control."""
+from __future__ import annotations
+
+from asyncio import sleep
+from typing import Any, cast
+
+from pyoverkiz.enums import OverkizCommand, OverkizCommandParam, OverkizState
+
+from homeassistant.components.climate import HVACMode
+from homeassistant.components.climate.const import PRESET_NONE
+from homeassistant.const import ATTR_TEMPERATURE
+
+from ..coordinator import OverkizDataUpdateCoordinator
+from .atlantic_pass_apc_heating_zone import AtlanticPassAPCHeatingZone
+from .atlantic_pass_apc_zone_control import OVERKIZ_TO_HVAC_MODE
+
+PRESET_SCHEDULE = "schedule"
+PRESET_MANUAL = "manual"
```

Comment:
Should we use `PRESET_COMFORT` instead?

Reply:
Manual is not the same as Comfort, which is another preset, but the device API don't allow the switch to it. I'la allow to virtually select preset comfort or eco on another PR by using eco or comfort temperatures (different than manual temperature).

Schedule can be either comfort or eco, depending on the time of the day (internal_schedule of the device).